Templates/CSharp/MSTestWinAppDriverCSharp/ProjectTemplates/CSharp/WinAppDriver/ScenarioTest.cs

```diff
@@ -0,0 +1,29 @@
+﻿using Microsoft.VisualStudio.TestTools.UnitTesting;
+using OpenQA.Selenium;
+using System.Threading;
+using System;
+
+namespace $safeprojectname$
+{
+    [TestClass]
+    public class ScenarioTest : TestSession
+    {
+        [TestMethod]
+        public void Test()
+        {
+            // Add the test logic
```

Comment:
Should we put some sample statement here ? Or just put these comments? @shyamnamboodiripad